bundles/org.openhab.binding.modbus.helioseasycontrols/README.md

```diff
@@ -0,0 +1,307 @@
+| sensorNameVoc4                   | sensorName                     | Sensor name - VOC 4                                              | String   |                                                                                                      |
+| sensorNameVoc5                   | sensorName                     | Sensor name - VOC 5                                              | String   |                                                                                                      |
+| sensorNameVoc6                   | sensorName                     | Sensor name - VOC 6                                              | String   |                                                                                                      |
+| sensorNameVoc7                   | sensorName                     | Sensor name - VOC 7                                              | String   |                                                                                                      |
+| sensorNameVoc8                   | sensorName                     | Sensor name - VOC 8                                              | String   |                                                                                                      |
+| softwareVersionBasis             | softwareVersionBasis           | Software version basis (format xx.xx)                            | String   |                                                                                                      |
+| operatingHoursSupplyAirVent      | operatingHours                 | Operating hours supply air fan (in minutes)                      | Number   | 0 - 2^32-1                                                                                           |
+| operatingHoursExtractAirVent     | operatingHours                 | Operating hours extract air fan (in minutes)                     | Number   | 0 - 2^32-1                                                                                           |
+| operatingHoursVhz                | operatingHours                 | Operating hours preheater (in minutes)                           | Number   | 0 - 2^32-1                                                                                           |
+| operatingHoursNhz                | operatingHours                 | Operating hours afterheater (in minutes)                         | Number   | 0 - 2^32-1                                                                                           |
+| outputPowerVhz                   | outputPower                    | Output power of preheater (in percent)                           | Number   | 0 - 2^32-1                                                                                           |

```

Comment:
According to the protocol specification, yes... but the spec is not great and I'm sure there are several mistakes... this might just be one of them...

Reply:
I was just wondering if there is some scaling expected as well... I have understood these details are often poorly documented